styles/footer.css

```diff
@@ -0,0 +1,101 @@
+*{
+    margin: 0%;
+}
+
+.footer-container{
+    background-color:#26272b;
+    padding:1% 0 0;
+    font-size:1rem;
+    height:auto;
+    color:#737373;
+    display: flex;
+}
+.directory-container{
+    height:100%;
+    width: 25%;
+
+}
+    .directory-img{
+        width: 200px;
+        padding:0 20px 0;
+
+    }
+    .directory-info{
+        color:white;
+        height: 70%;
+        padding:15px;
+        color:#737373;
+    }
+.about-us{
+    height: 100%;
+    width: 25%;
+    text-align: center;
+}
+    .about-us-title{
+        height: 30%;
+        color: white;
+        text-align: center;
+        margin:0;
```

Comment:
I think that this already had a margin 0
Because you added it on the 1st line
* {
margin: 0
}

Can you confirm if this is still working after you remove it?

Reply:
still working ! i fixed now